sdk/python/lib/test/automation/test_errors.py

```diff
@@ -28,7 +28,7 @@
 compilation_error_project = "compilation_error"
 runtime_error_project = "runtime_error"
 
-
+@pytest.mark.skipif('PULUMI_ACCESS_TOKEN' not in os.environ, reason="PULUMI_ACCESS_TOKEN not set")
```

Comment:
Nit: We tend to use `"` for strings rather than `'`, any reason not use `"PULUMI_ACCESS_TOKEN"`?

Reply:
Nope, fixing up the commit before merge.